src/app/shared/components/navbar/navbar.component.html

```diff
@@ -27,9 +27,13 @@
           }}</button>
       </li>
       <li class="nav-item">
-        <button onclick="window.location.href = '/team'" class="buttonNavbar" type="button">{{ 'NAVBAR.TEAM' | translate
-          }}</button>
+        <button class="buttonNavbar" type="button" [matMenuTriggerFor]="institutionalMenu">{{ 'NAVBAR.INSTITUTIONAL' | translate }}</button>
       </li>
+      <mat-menu #institutionalMenu="matMenu" xPosition="before" yPosition="below">
+        <button mat-menu-item onclick="window.location.href = '/team'">{{ 'NAVBAR.TEAM' | translate }}</button>
+        <button mat-menu-item onclick="window.location.href = 'https://csitba.web.app/'">CS</button>
+        <button mat-menu-item onclick="window.location.href = 'https://linktr.ee/wie_itba'">WIE</button>
+      </mat-menu>
```

Comment:
```suggestion
      <mat-menu #institutionalMenu="matMenu" xPosition="before" yPosition="below">
        <button mat-menu-item onclick="window.location.href = '/team'">{{ 'NAVBAR.TEAM' | translate }}</button>
        <button mat-menu-item onclick="window.location.href = 'https://csitba.web.app/'">CS</button>
        <button mat-menu-item onclick="window.location.href = 'https://linktr.ee/wie_itba'">WIE</button>
      </mat-menu>
      </li>
```
Me parece que de esta manera vas a evitar que se muestre por fuera de donde debería